packages/roosterjs-editor-plugins/lib/plugins/TableCellSelection/utils/updateSelection.ts

```diff
@@ -0,0 +1,17 @@
+import { IEditor } from 'roosterjs-editor-types';
+
+/**
+ * @internal
+ */
+export function updateSelection(
```

Comment:
I think we still need it. mostly on scenarios where the selection started after the end of the selection. If we use editor.select, the order of the selection is lost:

For example: 
![image](https://user-images.githubusercontent.com/8291124/232559560-44a88293-26b6-4484-a467-a0f21c53d48d.png)

A is start of selection and B is end of selection. 

If we wanted to select that text again we would need to do:
editor.select(B.node, B.offset, A.node, A.offset). 

And after that, if we press Shift + Left the result would be incorrect: Like this:
![image](https://user-images.githubusercontent.com/8291124/232560127-62f3b7df-9acd-4ab8-aeae-5e3438be8ab0.png)

Instead of expanding where the cursor was left of, we would set the start to be end of the selection.

Reply:
Please add comment to explain this. Otherwise it may be removed by someone else in the future.